e2e-tests/functions

```diff
@@ -36,6 +36,25 @@ create_namespace() {
 	fi
 }
 
+wait_for_delete() {
+	local res="$1"
+
+	echo -n "waiting for $res to be deleted"
+	set +o xtrace
+	retry=0
+	until (kubectl get $res || :) 2>&1 | grep NotFound; do
+		sleep 1
+		echo -n .
+		let retry+=1
+		if [ $retry -ge 120 ]; then
+			kubectl_bin logs ${OPERATOR_NS:+-n $OPERATOR_NS} $(get_operator_pod)
+			echo max retry count $retry reached. something went wrong with operator or kubernetes cluster
+			exit 1
+		fi
+	done
+	set_debug
```

Comment:
@gkech we do not have set_debug in PS.

Reply:
yeap, noticed it in Jenkins as well when executed OC test to verify this

![Screenshot 2025-05-27 at 4 56 08 PM](https://github.com/user-attachments/assets/959f1b51-2aed-4bfc-8251-0c205e3b5f2c)

will remove it